install/helm/agones/templates/extensions-deployment.yaml

```diff
@@ -114,6 +114,8 @@ spec:
               fieldPath: metadata.namespace
         - name: CONTAINER_NAME
           value: "agones-extensions"
+        - name: READINESS_SHUTDOWN_DURATION
+          value: {{ mul .Value.agones.extensions.readiness.periodSeconds .Value.agones.extensions.readiness.failureThreshold 2 | quote }}s
```

Comment:
This looks good. Let's also go ahead and configure `terminationGracePeriodSeconds` while we're at it - that way if someone configures things such that it's longer than the 30s `kubelet` gives us by default, we have a reasonable configuration. It can be configured for e.g. `periodSeconds * failureThreshold * 3` to give another safety envelope, but if we go beyond *2, I'd be shocked.